### PR TITLE
INDIdriver for the new direct drive mounts 'AOK Skywalker' from AOKSwiss/Magnetdrives

### PR DESCRIPTION
**Pull request by escribana**
Looking forward to receive your opinions.

### CODE REVIEW

File: indi-herkules/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+include_directories(${CMAKE_CURRENT_SOURCE_DIR})
+include_directories(${INDI_INCLUDE_DIR})
+include_directories(${NOVA_INCLUDE_DIR})
+
+include(CMakeCommon)
+
+set(HERKULES_VERSION_MAJOR 0)
+set(HERKULES_VERSION_MINOR 5)
```
Comment (knro) [inline indi-herkules/CMakeLists.txt]: Shouldn't minor be 1?

**Comment by escribana** [comment]
On Wed, 18 Sep 2019 07:52:05 -0700
Jasem Mutlaq <notifications@github.com> wrote:

> knro commented on this pull request.
> 
> 
> 
> > +LIST(APPEND CMAKE_MODULE_PATH
> > "${CMAKE_CURRENT_SOURCE_DIR}/cmake_modules/")  
> +LIST(APPEND CMAKE_MODULE_PATH
> "${CMAKE_CURRENT_SOURCE_DIR}/../cmake_modules/")
> +include(GNUInstallDirs) +
> +find_package(INDI COMPONENTS driver lx200 REQUIRED)
> +find_package(Nova REQUIRED)
> +
> +include_directories(${CMAKE_CURRENT_BINARY_DIR})
> +include_directories(${CMAKE_CURRENT_SOURCE_DIR})
> +include_directories(${INDI_INCLUDE_DIR})
> +include_directories(${NOVA_INCLUDE_DIR})
> +
> +include(CMakeCommon)
> +
> +set(HERKULES_VERSION_MAJOR 0)
> +set(HERKULES_VERSION_MINOR 5)
> 
> Shouldn't minor be 1?

... this is my "private" version number. ;-}

**Comment by knro** [comment]
The version number is actually fine. I thought I deleted this comment after I saw the changelog so disregard it. But the debian+documentation are still required :-)

**Comment by knro** [comment]
Hello, is there any update on the debian+documentation?

**Comment by escribana** [comment]
On Sat, 05 Oct 2019 22:30:00 -0700
Jasem Mutlaq <notifications@github.com> wrote:

> Hello, is there any update on the debian+documentation?

Yes, there are! I've made several improvments and begun to write the
documentation. But I was really tied up to my business for the last two
weeks, so the documents are not yet finished.

BTW, I'm in contact with the developers of the mount and they want to
rename the driver to "DDM Skywalker". Following their statement, the
driver I wrote will handle the whole product line of the
"Skywalker"-mounts (e.g. Herkules V12/V24/V48 and Atlas V24).

As soon as the documention is audited and the code is fully improved
and tested, I will put another pull request.

Regards
Antonio

**Comment by escribana** [comment]
The Debian files are uploaded (see https://github.com/indilib/indi-3rdparty/pull/1/commits/6243880c492053301e06ce13239023f30e8a3d74) and the requested douments are attached to this message.
The driver itself was tested during various nights and seems to work quite well! So I hope tit will make the way into the repo.

[indi-aok.zip](https://github.com/indilib/indi-3rdparty/files/3721559/indi-aok.zip)

**Comment by knro** [comment]
Thanks, that's great! Will upload documentation soon!

**Comment by knro** [comment]
I had to make a few changes to fix a few security warnings. Please check now.

Documentation is uploaded to website and launchpad builds updated to include the driver starting with the first nightly tomorrow.

**Comment by escribana** [comment]
All is ok!

The only thing I always stumbled upon is the Park button: The default naming still seems unintellegible to me. But after all it's peanuts!

**Comment by knro** [comment]
We can apply your suggestion to the INDI::Telescope class but we need some feedback from other developers on this.

**Pull request merged**